### ISSUE TITLE
Meeting minute pdf not displaying in one specific meeting

### ISSUE THREAD

**Issue by kevindherman** [proudcity/wp-proudcity#1963]
**Source**
#solanocountygis 

**Describe the bug**
If you click on the minutes button for Feb 16, 2022 it doesn't display the embedded pdf even though it does recognize that it has been added.
List view: https://regis.solanocounty.com/regis-members-meetings/
Meeting Detail minutes tab brings up the agenda instead: https://regis.solanocounty.com/meetings/regis-members-meeting-february-16-2022/#/tab-minutes

**To Reproduce**
1. Go to https://regis.solanocounty.com/regis-members-meetings/
2. Click on Feb 16, 2022 minutes
4. See error in that the minutes aren't there. 

**Expected behavior**
If you click minutes on any other meeting from this list you'll see the correct behavior. 
https://regis.solanocounty.com/regis-members-meetings/

**Screenshots**

List view
<img width="1053" alt="Screen Shot 2022-02-15 at 3 10 50 PM" src="https://user-images.githubusercontent.com/6032426/154141242-6bd810df-69d7-4dd9-86f3-af5041db92b1.png">

View without minutes
<img width="1287" alt="Screen Shot 2022-02-15 at 3 10 33 PM" src="https://user-images.githubusercontent.com/6032426/154141273-f9c571c4-42cb-46cf-9691-75cd91e5be54.png">

Minutes are attached correctly:
<img width="905" alt="Screen Shot 2022-02-15 at 3 11 02 PM" src="https://user-images.githubusercontent.com/6032426/154141292-4ff7fb27-06be-47cf-8bd3-accbd5240319.png">

**Comment by jlyon** [comment]
@kevindherman The screenshot above was from the minutes upload. We don't show minutes until after the event has concluded (https://github.com/proudcity/wp-proud-theme/blob/114d3db6a7d42634aa0cde288ce86f93ebd4c20f/templates/content-single-meeting.php#L201-L202).  I selected this file from the the Agenda Packet upload button and the file is now showing on the Agenda Packet tab. I also tweaked the location so it will not display anything if it is empty. Currently it's showing a comma right above the buttons (this will go out in the next release).
![Screenshot (78)](https://user-images.githubusercontent.com/550909/154152310-29ff2e48-7863-49f2-a631-877dc290205d.png)
![Screen Shot 2022-02-15 at 1 27 41 PM](https://user-images.githubusercontent.com/550909/154152322-36c48cc7-3b80-4701-b991-acdb2bf3f268.png)
